default.nix

```diff
@@ -46,11 +46,16 @@ let
       mkdir -p $out/bin $out/usr/bin $out/sbin
       ln -s ${stdenv.shell} $out/bin/sh
       ln -s ${coreutils}/bin/env $out/usr/bin/env
+      ln -s ${bashInteractive}/bin/bash $out/bin/bash
+      ln -s ${bashInteractive}/bin/bash $out/usr/bin/bash
 
       mkdir -p $out/etc
       echo '${passwd}' > $out/etc/passwd
       echo '${group}' > $out/etc/group
       echo '${nsswitch}' > $out/etc/nsswitch.conf
+      ln -s ${iana-etc}/etc/protocols $out/etc/
+      ln -s ${iana-etc}/etc/services $out/etc/
+      ln -s ${cacert}/etc/ssl $out/etc/
```

Comment:
libraries like openssl honor `NIX_SSL_CERT_FILE` which is set, this should't be necessary.